### PR TITLE
[MS-768] Social bar

### PR DESCRIPTION
**Pull request by stochna**

### CODE REVIEW

File: src/part/social-bar.tsx [hunk 1]
```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import { Locale } from "src/i18n/locale";
```
Comment (protyagov) [inline src/part/social-bar.tsx:2]: I think we do not need to import Locale here since it is not used

**Review by protyagov — APPROVED** [review]
All good, we could remove Locale from line 2 in social-bar since it is not used there.

File: src/part/social-bar.tsx (hunk 1 above)
Comment (stochna) [reply src/part/social-bar.tsx:2]: @protyagov oh, I forgot that I wanted to ask, probably we should add alt for links?

Comment (protyagov) [reply src/part/social-bar.tsx:2]: Right, would be good to add ALT links -- they help Google when they index things.

Comment (stochna) [reply src/part/social-bar.tsx:2]: I meant title, not alt

Comment (protyagov) [reply src/part/social-bar.tsx:2]: Right, something like this:
`alt={_("FOOTER.APPLE_BUTTON.ALT")} `

**Comment by stochna** [comment]
@protyagov I used titles instead of alt as alt is not allowed on anchor links. Also found and fixed one missing localization and renamed APPLE_BUTTON to APPLE_BTN as we always use "BTN" naming.

**Review by protyagov — APPROVED** [review]

**Pull request merged**